solr/core/src/test-files/solr/collection1/conf/solrconfig-parsing-update-processor-chains.xml

```diff
@@ -109,6 +109,21 @@
     </processor>
   </updateRequestProcessorChain>
 
+  <updateRequestProcessorChain name="parse-date-extraction-util-formats">
```

Comment:
Soon there will be no such "date extraction util" so I suggest another name.  Hmmm.  Perhaps "parse-date-patterns-from-extract-contrib".  Yeah it's long but no big deal.